<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunData</class>
 <widget class="QWidget" name="RunData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>156</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ALgorithm</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="lineAlgorithm"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="lineValue"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLineEdit" name="lineTime"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Iterations</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLineEdit" name="lineIterations"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
